launcher/ui/setupwizard/JavaWizardPage.cpp

```diff
@@ -69,6 +69,7 @@ bool JavaWizardPage::validatePage()
         case JavaSettingsWidget::ValidationStatus::AllOK:
         {
             settings->set("JavaPath", m_java_widget->javaPath());
+            break;
```

Comment:
are we sure this isn't supposed to fall through?

Either way this would need a return value